docs/partners/adopt.md

```diff
@@ -20,126 +20,184 @@ Adopting a pack has several advantages for the partner:
 ![pack example cyren](/doc_imgs/partners/packexample_cyren.png)
 
 ## Process
-The process to Adopt-a-Pack takes under 5 minutes! Follow the step by step instructions and watch the video tutorials below to learn how to submit your adoption pull requests.  
+Choose one of the options below and follow the step by step instructions or watch the video tutorials below to learn how to submit your adoption Pull Requests. 
 
-<strong>Visual Studio Instructions</strong>: If you are creating the pull request from Visual Studio, please follow the below steps. For additional guidance, watch the <a href="https://www.youtube.com/watch?v=9GPkhtRw4Oc">Visual Studio video overview</a>.
-<details>
-<summary>Click here to see the Visual Studio instructions</summary>
+<details><summary><strong>Working on a local clone of Content repo</strong></summary>
+<div>
 
 <br/>
+These methods assume that you have already [forked the `content` repository](https://xsoar.pan.dev/docs/tutorials/tut-setup-dev#step-2-fork-the-github-repo) and [cloned the fork onto your local machine](https://xsoar.pan.dev/docs/tutorials/tut-setup-dev#step-3-clone-the-github-fork-locally).
+Choose either one of the methods below to start or complete adoption.
 
-Please follow the below steps if you are working on a cloned Github repository from an IDE: 
+<br/>
+<details><summary><strong>Adopt Using Helper Script</strong></summary>
+
+<br/>
+
+<div>This script will automatically perform the necessary steps to create an adoption PR.
+
+**Note:** The script was only tested on Unix systems, specifically Mac OS and Ubuntu. It might not work correctly on other systems. 
```

Comment:
I don't relate to this disclaimer as it doesn't give anything to the user besides the feeling we are not taking responsibility if this fails. Instead, I think we should mention that the script is supported in Linux and Mac OS and if there's an issue the user should consult his Rep. from the PS team.